### DISCUSSION TITLE
Is there a Youtube button?

### DISCUSSION THREAD

**Discussion by BigB616** [aclap-dev/video-downloadhelper#277] [Help and Q&A]
### Your question

I am using Firefox.  I am wondering if there is a way to have a button that shows on Youtube. Vise having to go the add-on button.  Just seeing if there is a quick way of doing things.  Thanks for any help.

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\ravag",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by paulrouget** [comment]
What's wrong with the normal addon button?

**Reply by BigB616** [reply]
I have no button just the drop down add-on menu.  I was wondering if there was a way to get a button on the screen.

**Reply by paulrouget** [reply]
Did you pin the button? https://github.com/aclap-dev/video-downloadhelper/wiki/Lost-icon

**Reply by BigB616** [reply]
Thank you
